web_accounts_list.json

```diff
@@ -4591,6 +4591,73 @@
            "known_accounts" : ["john", "junech"],
            "category" : "dating",
            "valid" : true
+          },
+          {
+           "name" : "icq-chat",
+           "check_uri" : "https://icq.icqchat.co/members/{account}/",
+           "account_existence_code" : "200",
+           "account_existence_string" : "ICQ chat",
+           "account_missing_string" : "Oops! We ran into some problems",
+           "account_missing_code" : "404",
+           "known_accounts" : ["brookenora.54", "bigdaddy.77"],
+           "category" : "social",
+           "valid" : true
+          },
+          {
+           "name" : "HeyLink.me",
+           "check_uri" : "https://heylink.me/{account}/",
+           "account_existence_code" : "200",
+           "account_existence_string" : "Heylink.me",
+           "account_missing_string" : "We can't find the page that you're looking for :(",
+           "account_missing_code" : "404",
+           "known_accounts" : ["mohammed13", "johnny"],
+           "category" : "misc",
+           "valid" : true
+          },
+          {
+           "name" : "Yazawaj",
+           "check_uri" : "https://www.yazawaj.com/profile/{account}",
+           "account_existence_code" : "200",
+           "account_existence_string" : "أنا",
+           "account_missing_string" : "الصفحة المطلوبة غير موجودة",
+           "account_missing_code" : "404",

```

Comment:
I think you are testing these sites with a tool that follows redirects instead of showing them to you. For me, when I visit a non-existing username like `https://www.yazawaj.com/profile/eneUWTZD0UcMyLCHHU8G` the site redirects my browser with an HTTP 302 code instead of the 404 you have here. Can you test that more?

Reply:
Tested with an online tool [httpstatus ](https://httpstatus.io), got a 302 also, **will amend**.